job/src/process_results.py

```diff
@@ -33,7 +33,7 @@
                         ElectricUtilityInputs, ExistingBoilerOutputs, CHPOutputs, CHPInputs, \
                         ExistingChillerOutputs, CoolingLoadOutputs, HeatingLoadOutputs,\
                         HotThermalStorageOutputs, ColdThermalStorageOutputs, OutageOutputs,\
-                        REoptjlMessageOutputs
+                        REoptjlMessageOutputs, GHPInputs, GHPOutputs
```

Comment:
`GHPInputs` isn't used, but I think it should be.  We want it to update the inputs to `space_heating_efficiency_thermal_factor` and `cooling_efficiency_thermal_factor` after defaults are provided in Julia, like for CHP in `update_inputs_in_database` below - correct?

Reply:
Agreed, I did this in cab9a20ab1669a93c87d2ee08ea9a9d794c52c7c.

Reply:
Looks good, thank you!